subsys/bluetooth/controller/ll_sw/ull_adv_aux.c

```diff
@@ -51,6 +51,12 @@ static struct ll_adv_aux_set ll_adv_aux_pool[CONFIG_BT_CTLR_ADV_AUX_SET];
 static void *adv_aux_free;
 #endif /* (CONFIG_BT_CTLR_ADV_AUX_SET > 0) */
 
+#if defined(CONFIG_BT_CTLR_ADV_SET)
```

Comment:
If this define is duplicated in ull_adv.c file. Have this in ull_adv_internal.h file.

Reply:
Fixed.